frontend/src/pages/AdminHome.jsx

```diff
@@ -43,70 +52,69 @@ const AdminHome = () => {
   };
 
   const confirmDelete = async () => {
-    if (productIdToDelete) {
-      await axios.delete(`${import.meta.env.VITE_BACKEND_ENDPOINT_URL}/product/delete/${productIdToDelete}`)
-        .then(() => {
-          fetchProducts(); 
-          dispatch(setToast({
-            toast: {
-              message: 'Product has been successfully deleted!',
-              variant: 'success'
-            },
-            showToast: true
-          }))
-        })
-        .catch(error => console.error("Failed to delete product:", error));
-      setShowDeleteModal(false);
-      setProductIdToDelete(null);
+    if (!productIdToDelete) return;
+    try {
+      await axios.delete(`${import.meta.env.VITE_BACKEND_ENDPOINT_URL}/product/delete/${productIdToDelete}`);
+      fetchProducts();  // Refresh the list after deletion
+      dispatch(setToast({
+        message: "Product successfully deleted",
```

Comment:
update the message to product deleted successfully for consistency across all page